sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -50,7 +50,6 @@ class HiveSparkSubmitSuite
 
   override def beforeEach() {
     super.beforeEach()
-    System.setProperty("spark.testing", "true")
```

Comment:
is there any more places to remove?

Reply:
All the code was searched, and it was not found for the time being. thanks.